### ISSUE TITLE
Forum Thread errors

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#461]
**Reported by tstrunk on 1 May 37876603 21:20 UTC**
Rev. 14224 gives some error messages if you reply on the forums.

[forum_thread.php:99](source:trunk/boinc/html/user/forum_thread.php@14224#L99)
```
if (can_reply($thread, $logged_in_user)) {
should be:
if (can_reply($thread, $forum, $logged_in_user)) {
```

[forum.inc:587](source:trunk/boinc/html/inc/forum.inc@14224#L587)
```
if (($controls == FORUM_CONTROLS) && (can_reply($thread, $logged_in_user))) {
should be:
if (($controls == FORUM_CONTROLS) && (can_reply($thread, $forum, $logged_in_user))) {
```

**Comment by romw** [comment]
**Commented by Nicolas on 30 Sep 37877337 15:06 UTC**
Format and link to files.

**Comment by romw** [comment]
**Commented by davea on 14 Jun 37877819 03:06 UTC**
(In [14231]) - user web: fixes #482
